leases/tests/test_lease_models.py

```diff
@@ -270,3 +269,51 @@ def test_winter_storage_lease_cannot_update_place(
         winter_storage_lease.save()
 
     assert "Cannot change the place assigned to this lease" in str(exception.value)
+
+
+@freeze_time("2020-01-01T08:00:00Z")
+def test_berth_lease_is_active_before_season():
+    start_date = calculate_berth_lease_start_date()
+    lease = BerthLeaseFactory(status=LeaseStatus.PAID, start_date=start_date)
```

Comment:
isn't `calculate_berth_lease_start_date()` already a default for the `start_date`?

Reply:
Yeah, it is, just wanted to make it explicit, but I'll remove it. 👍